AudioConfigRP2040.h

```diff
@@ -36,6 +36,20 @@
 #define AUDIO_BITS 16   // available values are 8, 16, 24 (LEFT ALIGN in 32 bits type!!) and 32 bits
 // ****** END: These are define you may want to change. Best not to touch anything outside this range. ************/
 
+// The arduino-pico SDK has I2S Pins predefined for some boards.
+#if defined(ARDUINO_ARCH_RP2040)
+
+#if defined(PIN_I2S_BCLK)
+#define BCLK_PIN PIN_I2S_BCLK
+#define WS_PIN (PIN_I2S_BCLK+)
+#endif
+
+#if defined(PIN_I2S_DOUT)
+#define DOUT_PIN PIN_I2S_DOUT
+#endif
+
+#endif
+
```

Comment:
Is this erasing the config for all cases? When is this test going true? I would be against putting these pins at the default values of the arduino-pico library, with no easy change if that is the case. I think that (alongside the comment on the support of `ARDUINO_DATANOISETV_PICOADK`) this kind of config should be given by the board designer and not be hardcoded in Mozzi.